### ISSUE TITLE
Transparency

### ISSUE THREAD

**Issue by Simashax** [haxball/haxball-issues#173]
Custom teams' colors don't support transparency. Instead of transparency there is a white color.

**Comment by awareofdistractions** [comment]
Example: /colors red  333333 FFFFFFFF

**Comment by basro** [comment]
Transparency in the flash version was actually a bug, both in the custom stadium and team colors.
Since people made custom stadiums that depend on that bug I had to reimplement it there, but I'm not sure it's worth implementing for team colors.

**Comment by Simashax** [comment]
I think it's worth

**Comment by haxballer** [comment]
Transparent colors were nonsense and irritating. What makes sense if you want to have colorful shirts or not? Btw, I didn't know it was a bug, but it just looked like that.

**Comment by Simashax** [comment]
I don't agree with you, haxballer. The more options - the better, because sometimes you get used to the same stuff and you want changes. Transparency if used in a correct way makes the game more enjoyable, especially in certain maps. I guess this update would be easy to do and wouldn't take much space...

**Comment by haxballer** [comment]
But you're just about transparent balls, right? For you it is so important? There are many ideas and important elements, I just say that it doesn't make much sense, now you have many colors in this game, ok.

**Comment by Simashax** [comment]
Well... It doesn't make sense to you, but it makes sense to me. I already told why, ok?

**Comment by Leo-MapCreator** [comment]
@basro Is it hard to implement that for players? Cause that was kinda cool plus sometimes you use that for hide and seek map, where you hide over lines, with transparent uniform you could hide both in black lines and white lines (which were supposed to be under the player, exposing him, but since you are transparent you will be able to hide)

**Comment by basro** [comment]
I was not aware of the use @Leo-MapCreator mentioned, since it was required for that kind of game-mode then I'll be implementing this too.

**Comment by Leo-MapCreator** [comment]
What im saying is, its not VITAL, but we use it a lot, hence my map has the code for the transparent color in it's name

**Comment by helppe** [comment]
haxballer if we not add features that are not so important, haxball could look like this 

[https://i.ytimg.com/vi/ncB0ov5hT48/hqdefault.jpg](url)

**Comment by Leo-MapCreator** [comment]
xD
